#! /bin/sh /usr/share/dpatch/dpatch-run
## 32_check_ldap_pointer.dpatch by Jan Wagner <waja@cyconet.org>
## patch provided by dann frazier <dannf@debian.org>
##
## DP: Using deprecated interfaces of libldap API (bugs.debian.org/463322)
## DP: Upstream bug is: https://sourceforge.net/tracker/?func=detail&atid=397597&aid=1922579&group_id=29880

@DPATCH@
--- nagios-plugins-1.4.11.orig/plugins/check_ldap.c	2007-07-07 16:20:40.000000000 -0600
+++ nagios-plugins-1.4.11/plugins/check_ldap.c	2008-01-30 15:14:14.000000000 -0700
@@ -42,6 +42,7 @@
 #include "utils.h"
 
 #include <lber.h>
+#define LDAP_DEPRECATED 1
 #include <ldap.h>
 
 enum {
